### DISCUSSION TITLE
how are you supposed to use VSCode TS/JS language feature settings while using takeover mode?

### DISCUSSION THREAD

**Discussion by Tenrys** [vuejs/language-tools#3557] [Q&A] [answered]
![image](https://github.com/vuejs/language-tools/assets/3979239/5dc1bf97-8554-430e-a643-75b7937b53cc)
![image](https://github.com/vuejs/language-tools/assets/3979239/90c77659-ab45-4589-b4dd-7a16d74048d6)

these all become unknown configuration settings when using Takeover Mode for Volar. is there a different way to configure Volar to work with these settings? are you not meant to use Takeover Mode all the time?

**Comment by so1ve** [comment, marked as answer]
https://twitter.com/johnsoncodehk/status/1701092628413165593

Let's wait for volar 2.0 :)
